RIGS/models.py

```diff
@@ -501,6 +504,17 @@ class EventCrew(models.Model):
     notes = models.TextField(blank=True, null=True)
 
 
+@reversion.register
+class EventAuthorisation(models.Model, RevisionMixin):
+    event = models.OneToOneField('Event', related_name='authorisation')
+    email = models.EmailField()
+    name = models.CharField(max_length=255)
+    uni_id = models.CharField(max_length=10, blank=True, null=True, verbose_name="University ID")
```

Comment:
Despite even Ollie saying they did, having just asked Niki, they don't. This could be an issue.